linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_unique_name/rule.yml

```diff
@@ -19,10 +24,11 @@ references:
     cjis: 5.5.2
     disa: 770,804
     pcidss: Req-8.1.1
+    cis@rhel8: 6.2.17
 
 ocil_clause: 'a line is returned'
 
 ocil: |-
-    Run the following command to check for duplicate account names:
-    <pre>$ sudo pwck -qr</pre>
-    If there are no duplicate names, no line will be returned.
+    To ensure all accounts have unique names, run the following command:
```

Comment:
`s/To ensure/To verify/g`

Reply:
Rebased and fixed in: 586cedfb95523acbe0c0c92953851d6536c29230